Random_password_generator/README.md

```diff
@@ -0,0 +1,6 @@
+## Random Password Generator
+- This script can be used to generate a strong randomly password
+- The length of the password is passed as a Command Line argument
+
```

Comment:
Please add an example/sample use case for how to use this script.

Reply:
I have made the changes, please let me know if there is anything else I have to change